### PR TITLE
add step for installing visual studio preview

### PR DESCRIPTION
**Pull request by gkulin**
currently the preqreqs say user must have Visual Studio version later than 17.3.0 Preview 2. 

They need to have Visual Studio _Preview_ installed, which is a separate installation from Visual Studio. So, removing the prereq and adding installing Visual Studio Preview as the first step.

### CODE REVIEW

**Comment by gkulin** [comment]
Looks like I don't have permission to request reviewers so @savannahostrowski @alexwolfmsft would y'all mind taking a look at this?

**Review by savannahostrowski — APPROVED** [review]
LGTM for accuracy!

**Comment by Jak-MS** [comment]
@alexwolfmsft 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by alexwolfmsft — APPROVED** [review]

**Comment by alexwolfmsft** [comment]
#sign-off

**Review by jongio — APPROVED** [review]

File: articles/azure-developer-cli/includes/debug-visual-studio.md [hunk 1]
```diff
@@ -13,15 +13,18 @@ Run and debug apps built on your local machine using [Visual Studio](/visualstud
...
 - [Run `azd` with the Node.js template](../get-started.md)
 - [.NET SDK 6.0](https://dotnet.microsoft.com/download/dotnet/6.0)
-- [Visual Studio version later than 17.3.0 Preview 2 with the integration with `azd` feature flag enabled](/visualstudio/install/install-visual-studio)
 
-## Enable the preview feature
+## Install and enable the preview feature
+  
+1. Install [Visual Studio Preview](https://visualstudio.microsoft.com/vs/preview/) 
+   
+   Note: this is different from Visual Studio. If you have the non-preview version of Visual Studio, you still need to install this.
```
Comment (Court72) [inline articles/azure-developer-cli/includes/debug-visual-studio.md:21]:
```suggestion
   > [!NOTE]
   > This is different from Visual Studio. If you have the non-preview version of Visual Studio, you still need to install this. 
```

File: articles/azure-developer-cli/includes/debug-visual-studio.md [hunk 2]
```diff
@@ -13,15 +13,18 @@ Run and debug apps built on your local machine using [Visual Studio](/visualstud
...
 
-## Enable the preview feature
+## Install and enable the preview feature
+  
+1. Install [Visual Studio Preview](https://visualstudio.microsoft.com/vs/preview/) 
+   
+   Note: this is different from Visual Studio. If you have the non-preview version of Visual Studio, you still need to install this. 
 
-1. Open Visual Studio.
+2. Open Visual Studio Preview.
```
Comment (Court72) [inline articles/azure-developer-cli/includes/debug-visual-studio.md:23]:
```suggestion
1. Open Visual Studio Preview.
```

File: articles/azure-developer-cli/includes/debug-visual-studio.md [hunk 3]
```diff
@@ -13,15 +13,18 @@ Run and debug apps built on your local machine using [Visual Studio](/visualstud
...
 
-1. From the **Tools** menu, select **Options** > **Preview Features**.
+3. From the **Tools** menu, select **Options** > **Preview Features**.
```
Comment (Court72) [inline articles/azure-developer-cli/includes/debug-visual-studio.md:25]:
```suggestion
1. From the **Tools** menu, select **Options** > **Preview Features**.
```

**Pull request merged**